kevinene91/Fast-Food-Fast-API
# #160782572 database setup

### What does this PR do ? 
It setups a Postgresql database using psychopg2 
### Description of Task 
-install psychopg 
-Create a connection to the database 
-script to initialize tables after running app
### How to manually test this 
lone the repo
`git clone  https://github.com/kevinene91/Fast-Food-Fast-API.git`
Navigate to the folder 
`cd Fast-Food-Fast-Api`
switch to  this branch 
`git checkout  ch-database-setup-160782572`
Create virtualenv 
`virtualenv venv`
Activate the enviroment 
`source venv/bin/activate`
install requirements 
`pip install -r requirements.txt`
create databse with postgres
export enviroment variables for database
`python run.py`
check databse tables
### Related PVT story 
[160782572](https://www.pivotaltracker.com/n/projects/2193832/stories/160782572)
